"xcode/test_get_simulators" integration tests fail in windows

I'm running pytest on all tests in the "test" folder on branch master and the tests in xcode/test_get_simulators are failing.

For example:
The test "test_single_iOS_runtime" fails with the following stack_trace:
```
FAILED [ 83%]
tests\integrations\xcode\test_get_simulators.py:78 (test_single_iOS_runtime)
simulator = 'C:\\Users\\saroa\\AppData\\Local\\Temp\\pytest-of-saroa\\pytest-33\\test_single_iOS_runtime0\\CoreSimulator.framework'

    def test_single_iOS_runtime(simulator):
        "If an iOS version is installed, devices can be found"
        mock_input = mock.MagicMock()
    
        sub = mock.MagicMock()
>       sub.check_output.return_value = simctl_result('iOS-13.2-only')

integrations\xcode\test_get_simulators.py:84: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
integrations\xcode\test_get_simulators.py:23: in simctl_result
    return f.read()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <encodings.cp1255.IncrementalDecoder object at 0x0000024D6A3922B0>
input = b'{\r\n  "devicetypes" : [\r\n    {\r\n      "name" : "iPhone 4s",\r\n      "bundlePath" : "\\/Applications\\/Xcode.ap...FD7E18C83934",\r\n        "state" : "Shutdown"\r\n      },\r\n      "state" : "(unavailable)"\r\n    }\r\n  }\r\n}\r\n'
final = True

    def decode(self, input, final=False):
>       return codecs.charmap_decode(input,self.errors,decoding_table)[0]
E       UnicodeDecodeError: 'charmap' codec can't decode byte 0xca in position 4618: character maps to <undefined>

..\..\..\AppData\Local\Programs\Python\Python37\lib\encodings\cp1255.py:23: UnicodeDecodeError
```

Fixing it.

--- a/tests/integrations/xcode/test_get_simulators.py
+++ b/tests/integrations/xcode/test_get_simulators.py
@@ -19,7 +19,7 @@ def simulator(tmp_path):
 def simctl_result(name):
     """Load a simctl result file from the sample directory, and return the content"""
     filename = Path(__file__).parent / 'simctl' / '{name}.json'.format(name=name)
-    with filename.open() as f:
+    with filename.open(encoding="utf-8") as f:
         return f.read()
 
 

